src/main/scala/diplomaticobjectmodel/model/OMPerformanceMonitor.scala

```diff
@@ -11,8 +11,8 @@ case class OMPerformanceMonitor(
   _types: Seq[String] = Seq("OMPerformanceMonitor", "OMComponent", "OMCompoundType")
 ) extends OMComponent
 
-object PerformanceMonitor {
-  def permon(coreParams: RocketCoreParams): Option[OMPerformanceMonitor] = {
+case object PerformanceMonitor {
```

Comment:
Why did you change this to a `case object`? Is there anything that you need from the `case`ness?